keyboards/boardwalk/keymaps/nchristus/keymap.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "boardwalk.h"
+#include "action_layer.h"
+
+// Layer shorthand
+#define _QW 0
+#define LOWER F(1)
+#define RAISE F(2)
+
+#define CTRLESC CTL_T(KC_ESC)
+#define XXXXXXX KC_NO
```

Comment:
```suggestion
```

This shortcut is already defined in core, so you don't need it here :)

Reply:
Fixed with https://github.com/qmk/qmk_firmware/pull/5195/commits/f821eaf3e7285031370155a41a748a022153752b